distribution/rules.bzl

```diff
@@ -175,7 +283,7 @@ distribution = rule(
             default="//distribution:archiver.py"
         )
     },
-    implementation = _distribution_impl,
+    implementation = _old_distribution_impl,
```

Comment:
apparently `_old_distribution_impl` is used here. why is it not just called `distribution_impl`?